server/src/rbx_snapshot.rs

```diff
@@ -194,6 +195,21 @@ pub fn snapshot_project_tree<'source>(
     snapshot_project_node(context, imfs, &project.tree, Cow::Borrowed(&project.name))
 }
 
+pub fn infer_class_name(
+    instance_name: Cow<str>,
+) -> Option<Cow<str>> {
+    match get_class_descriptor(&instance_name) {
```

Comment:
```
match get_class_descriptor(&instance_name) {
    Some(instance) if instance.is_service() => Some(instance_name),
    _ => None,
}
```

Reply:
I did not know about this syntax! Thanks for the tip